src/practical_settings/GPG.tex

```diff
@@ -2,30 +2,43 @@
 \gdef\currentsectionname{GPG}
 \gdef\currentsubsectionname{GnuPG}
 
-The OpenPGP protocol
-\footnote{\url{https://tools.ietf.org/search/rfc4880}}
- uses asymmetric encryption to protect a session key which is used to encrypt a message. Additionally, it signs messages via asymmetric encryption and hash functions. %% what? - azet
-Research on SHA-1 conducted back in 2005\footnote{\url{https://www.schneier.com/blog/archives/2005/02/sha1\_broken.html}} has made clear that collision attacks are a real threat to the security of the SHA-1 hash function. PGP settings should be adapted to avoid using SHA-1.
+The OpenPGP protocol\footnote{\url{https://tools.ietf.org/search/rfc4880}} defines a set of asymmetric- and symmetric encryption algorithms, signature methods and compression protocols. GnuPG\footnote{\url{https://gnupg.org/}}, a FOSS implementation of the OpenPGP standard, is widely used for mail encryption.
+ 
+GnuPG signs a message (SHA-2, RIPEMD or SHA-1), encrypts it symmetrically (AES, CAMELLIA, TWOFISH, BLOWFISH, 3DES, CAST5 or IDEA) and encrpts the symmetric key and the hash with Bob's public key asymmetrically (RSA, ELG, DSA, ECDH, ECDSA or EDDSA).
```

Comment:
Looks good, but; I don't think it's necessary to point out every single available algorithm in these paragraphs. Also: these may change in the future. ECDH, ECDSA/EdDSA are experimental right now. You cannot even upload them to keyservers, they are also incompatible with version 1 of GnuPG, which many people still use.